lib/node_modules/@stdlib/math/base/complex/divide/test/test.js

```diff
@@ -0,0 +1,283 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var isinfinite = require( '@stdlib/math/base/assert/is-infinite' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var pow = require( '@stdlib/math/base/special/pow' );
+var EPS = require( '@stdlib/math/constants/float64-eps' );
+var toBinaryString = require( '@stdlib/math/base/utils/float64-to-binary-string' );
+var cdiv = require( './../lib' );
+
+
+// FIXTURES //
+
+var quotients = require( './fixtures/julia/quotients.json' );
+
+
+// HELPERS
```

Comment:
Yeah, I was just thinking about this. This is fine for now. A similar approach would be to compute the Hamming distance, but a numeric approach would be to measure number of ULP. Unfortunately, we have yet to write the ULP distance function. On the todo, but have to yet to find the time. Your current approach can be revisited. Just include a `TODO: revisit once ULP distance fcn is written` in the fixture file.